site/docs/v1/tech/tutorials/integrate-angular.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: doc
+title: Integrate Your Angular Application With FusionAuth
+description: Integrate your Angular application with FusionAuth
+navcategory: getting-started
+prerequisites: nodejs
+technology: Angular
+language: TypeScript
+---
+
+{% include docs/integration/_intro.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md language=page.language %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but we are going to use the {{page.language}} client library. The instructions below use `npm` on the command line, but you can use the client library with an IDE of your preference as well.
+
+First, make a directory:
+
+```shell
+mkdir setup-fusionauth && cd setup-fusionauth
+```
+
+Now, copy and paste the following code into `package.json`.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/package.json %}
+```
+
+Now you need to install the dependencies in `package.json`.
+
+```shell
+npm install
+```
+
+Then copy and paste the following code into `setup.js`. This file uses the [FusionAuth API](/docs/v1/tech/apis/) to configure an Application, CORS, and more to allow for easy integration.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/sonderformat-llc/fusionauth-example-client-libraries/angular/typescript/setup-angular.js %}
+```
+
+Then, you can run the setup script.
+
+```shell
+fusionauth_api_key=YOUR_API_KEY_FROM_ABOVE npm run setup
+```
+
+{% include _callout-note.liquid content="The setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than `Default`. To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. <br><br> Refer to the [Typescript client library](/docs/v1/tech/client-libraries/typescript) documentation for more information." %}
+
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key='YOUR_API_KEY_FROM_ABOVE'
+npm run setup
+```
+
+If you want, you can [log into your instance](http://localhost:9011) and examine the new Application the script created for you.
+
+## Create Your {{page.technology}} Application
+
+Now you are going to create a {{page.technology}} application. While this section uses a simple {{page.technology}} application, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
```

Comment:
```suggestion
Now you are going to create an {{page.technology}} application. While this section uses a simple {{page.technology}} application, you can use the same steps to integrate any {{page.technology}} application with FusionAuth.
```

Reply:
Or maybe we could rephrase this sentence to prevent these kind of things in the future when we copy and paste? E.g. _"Now you are going to create a simple {{page.technology}} application"_